tests/test_PostProcessWorkflow.py

```diff
@@ -73,6 +74,7 @@ def get_status_errors(self, cursor, datarun_id):
                 errors.append(result)
         return errors
 
+    @unittest.skip("Skipping due to catalog start message not sending in CI")
```

Comment:
Can you change this to [pytest](https://docs.pytest.org/en/6.2.x/skipping.html#skipping-test-functions)?